tmt/utils.py

```diff
@@ -3547,14 +3546,7 @@ def _patch_plan_schema(schema: Schema, store: SchemaStore) -> None:
             ]
 
         schema['properties'][step] = {
-            'oneOf': refs + [
-                {
-                    'type': 'array',
-                    'items': {
-                        'anyOf': refs
-                        }
-                    }
-                ]
+            'oneOf': [*refs, {'type': 'array', 'items': {'anyOf': refs}}]
```

Comment:
Reformatting feels definitely like a step back, the visual hints WRT structure is lost.

Reply:
Keeping the original formatting, at least as much as autopep8 allows.